recipes/nginx.rb

```diff
@@ -0,0 +1,45 @@
+#
+# Cookbook Name:: cachethq
+# Recipe:: nginx
+#
+# All rights reserved - Do Not Redistribute
+#
+
+cachethq = node.default['cachethq']
+nginx = cachethq['nginx']
+fpm = cachethq['php-fpm']
+
+# somehow apache2 is also installed
```

Comment:
it is better to give explanation why you need to remove nginx, ie. `to avoid duplicate web server, port blocked because already used, etc`

Reply:
done.